### DISCUSSION TITLE
Nextjs 13 and RSC

### DISCUSSION THREAD

**Discussion by leognmotta** [heroui-inc/tailwind-variants#32] [General]
Hello, I have been experimenting with Tailwind Variants. Since Next.js 13 announced its move into RSC, we anticipate a growing demand for UI libraries that are runtime-free and offer flexibility for easily overriding or customizing styles. I believe this project will empower us to build amazing customizable and runtime-free libs on top of Tailwind. 
Congratulations on this innovative initiative!

**Comment by 2khan** [comment]
Couldn't agree more! Great time to be a front-end developer, thanks to `tailwind-variants`.

**Comment by Code-Victor** [comment]
It's awesome to have a runtime free alternative
